bibliopixel/remote/control.py

```diff
@@ -104,13 +110,32 @@ def __init__(self, layout, animations,
         self.handlers = {
             'run_animation': self.run_animation,
             'stop_animation': self.stop_animation,
-            'get_config': self.get_config
+            'get_config': self.get_config,
```

Comment:
Hahaha... I originally had that. And the reason I didn't do that here is that trigger_animation just calls run_animation... the reason I did this is run_animation pushes back to a send queue, trigger_animation does not. I didn't want the triggers have to care about getting a response from a queue, so that's why there's all the mangling of do I send a response or not?